client/components/Cart.js

```diff
@@ -63,48 +66,39 @@ export class Cart extends Component {
-        {cart
-          //added a cart sort so that items would be redrawn in same order when changing quantity
-          .sort((a, b) => (b.artworkId > a.artworkId ? -1 : 1))
-          .map((cartItem) => {
-            const cartItemArtwork = artworks.find(
-              (art) => art.id === cartItem.artworkId
-            )
-            const exceedsArtworkQuantity =
-              cartItem.quantity > cartItemArtwork.quantity
-            return (
-              <div key={cartItem.artworkId}>
-                <h4>{cartItemArtwork.title}</h4>
-                <h5>
-                  <i>{cartItemArtwork.artist.name}</i>
-                </h5>
-                {exceedsArtworkQuantity ? (
-                  <h5 className="noQty">
-                    You've added more to your cart ({cartItem.quantity}) than is
-                    currently in stock! Please choose a revised quantity.
-                  </h5>
-                ) : null}
-                <h5>
-                  Quantity:
-                  <select
-                    id="qty"
+      <div className="all-cart">
+        <div className="cart-items">
+          { cart
+            //added a cart sort so that items would be redrawn in same order when changing quantity
+            .sort((a,b) => b.artworkId > a.artworkId ? -1 : 1)
+            .map(cartItem => {
+              const cartItemArtwork = artworks.find(art => art.id === cartItem.artworkId)
+              const exceedsArtworkQuantity = cartItem.quantity > cartItemArtwork.quantity
+              return (
+                <div key={cartItem.artworkId} className="cart-item">
+                  <h4>{ cartItemArtwork.title }</h4>
+                  <h5><i>{ cartItemArtwork.artist.name }</i></h5>
+                  { exceedsArtworkQuantity ?
+                    <h5 className="noQty">You've added more to your cart ({cartItem.quantity}) than is currently in stock! Please choose a revised quantity.</h5>
+                  : null
+                  }
+                  <h5>
+                    Quantity:
+                  <select id="qty"
                     defaultValue={cartItem.quantity}
-                    onChange={(ev) =>
-                      this.changeQuantity(ev.target.value, cartItem)
-                    }>
+                    onChange={(ev)=>this.changeQuantity(ev.target.value, cartItem)}
+                  >
                     {this.quantitySelect(cartItemArtwork)}
                   </select>
-                </h5>
-                <button type="button" onClick={() => this.removeItem(cartItem)}>
-                  Remove from cart
-                </button>
-                <hr />
-              </div>
-            )
-          })}
+                  </h5>
+                  <button onClick={()=>this.removeItem(cartItem)}>Remove from cart</button>
+                  <hr />
+                </div>
+              )
+            })}
+        </div>
+        <Checkout cart={ cart } artworks={ artworks } />
```

Comment:
I think we should have some logic here that prevents checkout if you have an invalid qty in your cart.